tests/regression/facebook/test_chrome_facebook_ft_type_composerbox_100_txt.sikuli/test_chrome_facebook_ft_type_composerbox_100_txt.py

```diff
@@ -0,0 +1,51 @@
+# if you are putting your test script folders under {git project folder}/tests/, it will work fine.
+# otherwise, you either add it to system path before you run or hard coded it in here.
+sys.path.append(sys.argv[2])
+import os
+import common
+import facebook
+import shutil
+import browser
+import time
+
+# Disable Sikuli action and info log
+com = common.General()
+com.infolog_enable(False)
+
+chrome = browser.Chrome()
+fb = facebook.facebook()
+
+chrome.clickBar()
+chrome.enterLink(sys.argv[3])
+fb.wait_for_loaded()
+
+sleep(2)
+setAutoWaitTimeout(10)
```

Comment:
`setAutoWaitTimeout()` is not useful if there is no `wait()`
Please refer to #639 for further information.